### ISSUE TITLE
When a request is "in-progress" have a follow up button.

### ISSUE THREAD

**Issue by JosephMcArthur** [OAButton/discussion#680]
Thought I had more time than I did to write this, but leaving a note to myself to expand.

**Comment by JosephMcArthur** [comment]
Okay, so, when things are in progress the only real action is to follow up. 

48 hours after sending we should show a button to users allowing them to follow up. This:
* sends emails/author_followup_one.txt

to an author. 

- [ ] first we need to write emails/author_followup_one.txt. Assigning @chealsye first as we have already done drafts of this elsewhere.

**Comment by chealsye** [comment]
@JosephMcArthur What exactly is this doing? 48 hours after we email the author, we are allow the user to follow up? Can you explain more?

**Comment by JosephMcArthur** [comment]
This is just for admins, it's just trying to make the follow-up process easier.

**Comment by chealsye** [comment]
All right - to confirm you want a draft of a follow up email to the author that will be able to be sent 48 hours or later from any admin (Natalia, other OAB, volunteers)? 

I think 48 hours is a very small window and following up with someone in that time span (especially if the initial request came on a weekend, a busy week they had, their own national holidays, but basically really anything to do with them being a busy, real person) will mostly serve to annoy authors. I understand wanting to increase the turn around time, but 48 hours is a very small window for a busy faculty member to respond. Is there evidence from @svmelton's interviews to support a 48 hour follow up? If you insist on 48 hours I'm going to write a super friendly and super short email, because I'm worried.

**Comment by natalianorori** [comment]
I agree with Chealsye on this one. 48 hours might be to soon to start following up, and some authors might get annoyed by this and consider it spammy (specially those who might get a lot of requests from us in the future)

**Comment by JosephMcArthur** [comment]
I've seen data in the past that suggests an email like this is more than likely never going to be responded to after 24 hours in someone's inbox. I can't track down that data for the life of me now. Given users will want papers as close to instantly as possible, waiting longer seems painful. 

Have asked @svmelton to ask our next batch of authors about this in user testing. @chealsye I'd hold off writing for now.

Note to self, if we don't optimise hours after sending, should we optimize for day of the week?
